### DISCUSSION TITLE
[Feature Request][frameit] Support new iPhone 12 Family

### DISCUSSION THREAD

**Discussion by fdstevex** [fastlane/fastlane#17565] [Ideas]
One of the required resolutions now is 1284x2778 (the iPhone 12 Pro Max resolution).  I have screenshots with this resolution in my screenshots dir along wth the rest, but when I run `fastlane frameit`, it's not framing them.

`Unsupported screen size [1284, 2778] for path './screenshots/en-US/en-6.5_1_Main.png'`

<img width="456" alt="CleanShot 2020-11-06 at 07 32 53@2x" src="https://user-images.githubusercontent.com/4220256/98366679-5f8c8480-2002-11eb-8cae-546d854103ec.png">

Looks like Fastlane uses the Facebook device frames (https://www.facebook.design/devices) and they haven't been updated since February. 

Anyone have frames they're using with frameit for the iPhone 12?

**Comment by rogerluan** [comment]
Ah this is a 100% valid issue. Looks like we'll have to add support to the new 4 devices announced in September: iPhone 12 Mini, iPhone 12, iPhone 12 Pro, iPhone 12 Pro Max. Looks like this should be done in `frameit/lib/frameit/device_types.rb`, however I'm not sure where the actual frame information come from (in the code).

This is definitely something that we'll need to address soon enough 😱

**Comment by futuretap** [comment]
Where the frame info comes from? `fastlane frameit download_frames` downloads the offsets from `https://fastlane.github.io/frameit-frames/latest/offsets.json`.

Hope this will be fixed soon.

**Comment by rogerluan** [comment]
This might fix it https://github.com/fastlane/frameit-frames/pull/15
